README.md

```diff
@@ -1,35 +1,492 @@
-What is Swrve
--------------
-Swrve is a single integrated platform delivering everything you need to drive mobile engagement and create valuable consumer relationships on mobile.  
+Swrve
+=
+Swrve is a single integrated platform delivering everything you need to drive mobile engagement and create valuable consumer relationships on mobile.
+
 This Unity3D SDK will enable your app to use all of these features.
 
+Table of Contents
+-
+
+- [Getting started](#getting-started)
+  - [Requirements](#requirements)
+    - [Unity 4.0.0+](#unity-400)
+    - [Unity 5+ Personal or Unity 4.0.0+ Pro](#unity-5-personal-or-unity-400-pro)
+- [Installation Instructions](#installation-instructions)
+  - [In-app Messaging](#in-app-messaging)
+    - [In-app Messaging Deeplinks](#in-app-messaging-deeplinks)
+  - [iOS Push Notifications](#ios-push-notifications)
+    - [Creating and uploading your iOS Cert](#creating-and-uploading-your-ios-cert)
+    - [Provisioning your app](#provisioning-your-app)
+  - [Android Push Notifications](#android-push-notifications)
+    - [Using a custom `GcmReceiver`](#using-a-custom-gcmreceiver)
+    - [Custom `MainActivity`](#custom-mainactivity)
+    - [No custom `MainActivity`](#no-custom-mainactivity)
+    - [Advanced Android Push](#advanced-android-push)
+  - [Sending Events](#sending-events)
+    - [Sending Named Events](#sending-named-events)
+    - [Send User Properties](#send-user-properties)
+    - [Sending Virtual Economy Events](#sending-virtual-economy-events)
+    - [Sending IAP Events and IAP Validation](#sending-iap-events-and-iap-validation)
+  - [Integrating Resource A/B Testing](#integrating-resource-ab-testing)
+  - [Testing your integration](#testing-your-integration)
+- [Upgrade Instructions](#upgrade-instructions)
+- [How to run the demo](#how-to-run-the-demo)
+- [How to build the SDK](#how-to-build-the-sdk)
+- [Contributing](#contributing)
+- [License](#license)
+
+
 Getting started
----------------
-Have a look at the quick integration guide at http://docs.swrve.com/developer-documentation/37926233/
+=
 
 Requirements
-------------
+-
+Swrve supports Unity version 4.0.0 or later on Android and iOS only. **Swrve Unity SDK is not currently supported on Unity Web Player or Windows 8.**
+
 ### Unity 4.0.0+
 The SDK needs a Unity version higher than 4.0.0 to be able to compile.
 
 ### Unity 5+ Personal or Unity 4.0.0+ Pro
 Some of the features like push notifications require the use of native plugins. Native plugins are now available in Unity Personal 5+ or previvous versions of Unity Pro.
 
+Installation Instructions
+=
+1. Download the [latest release](https://github.com/Swrve/swrve-unity-sdk/releases/latest)
```

Comment:
@Sergio-Mira the 3.5 release is not here. Could uploading that become part of your deployment process?